tanip_pipeline.conf

```diff
@@ -10,27 +10,26 @@
     Path    ./app/logfile.log
     Parser  json
     Tag     logfile1
+    
+[INPUT]
+    Name    tail
+    Path    ./app/another_logfile.log
```

Comment:
Should this path match up with what is being produced by the generator app?

Reply:
Yes it does on my end, I think I discarded the change for git so that it didn't get cluttered with other files.